anomalib/models/ganomaly/config.yaml

```diff
@@ -0,0 +1,120 @@
+dataset:
+  name: mvtec
+  format: mvtec
+  path: ./datasets/MVTec
+  url: ftp://guest:GU.205dldo@ftp.softronics.ch/mvtec_anomaly_detection/mvtec_anomaly_detection.tar.xz
+  category: bottle
+  task: classification
+  label_format: None
+  tiling:
+    apply: false
+    tile_size: null
+    stride: null
+    remove_border_count: 0
+    use_random_tiling: False
+    random_tile_count: 16
```

Comment:
I think tiling should be on by default. The encoder and decoder network in this model is capable of handling 64x64 images max. So we need to use either 32x32 or 64x64 windows.